apps/docs/pages/docs/api-reference/components/puck.mdx

```diff
@@ -201,6 +202,31 @@ An array of histories to reset the Puck state history state to.
 
 The index of the histories to set the user to.
 
+### `onAction(action, appState, prevAppState)`
+
+Callback that triggers when Puck dispatches an [action](https://puckeditor.com/docs/api-reference/actions), like as `insert` or `set`. Use this to track changes, perform side effects, or sync with external systems.
```

Comment:
Sorry, I made a typo in the above. Would you be able to push one more fix?

```suggestion
Callback that triggers when Puck dispatches an [action](https://puckeditor.com/docs/api-reference/actions), like `insert` or `set`. Use this to track changes, perform side effects, or sync with external systems.
```

Reply:
Sure 😊 I should have checked it.